src/main/java/fibon.java

```diff
@@ -0,0 +1,15 @@
+
+
+public class fibon {
+
+    public static int fi(int num) {
```

Comment:
```suggestion
    public static int computeFibonacci(int num) {
```

a method should be a verb if possible